.npmignore

```diff
@@ -0,0 +1 @@
+*.story.js
```

Comment:
Are we no longer publishing from `dist` by copying the `package.json` over or why did you feel that we need this? The stories shouldn't end up there in any case?

Reply:
Yes, we don't need to "publish from dist" anymore. It's just that we have some `*.story.js` files in the `materials` folder and I thought it would be good to not publish those

Reply:
Thanks for explaining. I googled a bit and this seems like the best option after all.